DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_SearchResultsCount.cshtml

```diff
@@ -5,5 +5,5 @@
   @{
     var count = Model.MatchingSearchResults;
   }
-  <span role="alert" id="results-count">@count matching @(count == 1 ? "result" : "results")</span>
+  <span role="alert" aria-live="polite" id="results-count">@count matching @(count == 1 ? "result" : "results")</span>

```

Comment:
I originally replaced the `role="alert"` with `aria-live="polite" `as it seems more appropriate given the description in the docs:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
However, we need the `role="alert"` to announce the results when there is no javascript enabled. Seeing as how `role="alert" ` implements the same functionality as the `aria-live="polite"`, should I just get rid of the new addition and keep this how it was?

Reply:
If this doesn't change the ux at all, I don't think we should bother adding it. Extra code to understand and maintain.